pom.xml

```diff
@@ -42,6 +42,8 @@
 
     <properties>
         <jenkins.version>1.580.1</jenkins.version>
+        <java.level>6</java.level>
+        <findbugs.failOnError>false</findbugs.failOnError>
```

Comment:
failOnError is set to false because I don't want the build to be marked as failed, rather unstable if there are any findbugs warnings, so it should be kept as false.